cilium/CFP-34577-per-service-lb-algorithm-selection.md

```diff
@@ -0,0 +1,118 @@
+# CFP-34577: Per-service Load Balancing Algorithm Selection
+
+**SIG: SIG-DATAPATH**
+**Begin Design Discussion: 2024-08-28**
+**Cilium Release:** 1.17
```

Comment:
If you think this will be implemented prior to 1.17 feature freeze, feel free to keep this as-is, but given the expected freeze timeline in the next couple of weeks ([post](https://github.com/cilium/cilium/discussions/35813)), 1.18 seems more likely:

```suggestion
**Cilium Release:** 1.18
```

Reply:
I talked with @borkmann and we are aiming to mere the PR (without the change in StateDB/experimental) till freeze